build fails

**How can we reproduce this bug?**

1. `git clone git@github.com:zurb/foundation-sites.git`
2. `cd foundation-sites`
3. `gem install scss-lint && npm install`
4. `npm run start`

**What did you expect to happen?**
For the docs sites to work

**What happened instead?**
```
marc@rey ~/src/foundation-sites (develop) npm start

> foundation-sites@6.2.3 start /Users/marc/src/foundation-sites
> gulp

[11:21:54] Using gulpfile ~/src/foundation-sites/gulpfile.js
[11:21:54] Starting 'clean'...
[11:21:54] Finished 'clean' after 11 ms
[11:21:54] Starting 'copy'...
[11:21:54] Finished 'copy' after 12 ms
[11:21:54] Starting 'docs:all'...
[11:21:54] Starting 'sass:foundation'...
[11:21:54] Starting 'sass:docs'...
[11:21:54] Starting 'javascript:foundation'...
[11:21:54] Starting 'javascript:deps'...
[11:21:54] Starting 'javascript:docs'...
[11:21:54] Starting 'watch'...
[11:21:54] Finished 'watch' after 156 ms
[11:21:56] Finished 'sass:docs' after 2.11 s
[11:21:56] Supercollider: processed abide.html in 0.51 s with sass, js
[11:21:56] Supercollider: processed accessibility.html in 0.56 s
[11:21:57] Finished 'javascript:deps' after 2.55 s
fs.js:640
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/marc/src/foundation-sites/docs/pages/assets/css/docs.css'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at Object.self.MD5 (/Users/marc/src/foundation-sites/node_modules/cachebust/lib/cachebust.js:35:23)
    at Object.exports.busted (/Users/marc/src/foundation-sites/node_modules/cachebust/lib/cachebust.js:58:40)
    at /Users/marc/src/foundation-sites/node_modules/gulp-cache-bust/index.js:44:40
    at /Users/marc/src/foundation-sites/node_modules/graceful-fs/graceful-fs.js:78:16
    at tryToString (fs.js:455:3)
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:442:12)

npm ERR! Darwin 16.0.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.8.1
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! foundation-sites@6.2.3 start: `gulp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the foundation-sites@6.2.3 start script 'gulp'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the foundation-sites package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs foundation-sites
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls foundation-sites
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/marc/src/foundation-sites/npm-debug.log
```
**Versions:**
```
marc@rey ~/src/foundation-sites (develop) node -v
v6.8.1

marc@rey ~/src/foundation-sites (develop) npm -v
3.10.8

marc@rey ~/src/foundation-sites (develop) ruby -v
ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin16]

marc@rey ~/src/foundation-sites (develop) gem -v
2.5.1
```

--- a/package.json
+++ b/package.json
@@ -43,7 +43,7 @@
     "gulp-add-src": "^0.2.0",
     "gulp-autoprefixer": "^2.3.1",
     "gulp-babel": "^6.1.1",
-    "gulp-cache-bust": "^1.0.2",
+    "gulp-cache-bust": "1.0.2",
     "gulp-concat": "^2.4.3",
     "gulp-cssnano": "^2.1.0",
     "gulp-eslint": "^2.0.0",

